defaults/main.yml

```diff
@@ -9,7 +9,7 @@ cryptpad_identifier: cryptpad
 cryptpad_uid: ''
 cryptpad_gid: ''
 
-cryptpad_version: 5.2.1
+cryptpad_version: version-2024.9.1
```

Comment:
It's better to keep `cryptpad_version` simple (e.g. `2024.12.0`, not `version-2024.12.0`), so make it match the git tag - [2024.12.0](https://github.com/cryptpad/cryptpad/tree/2024.12.0), so that self-building (see `cryptpad_container_image_self_build`) can work.

You can massage this simple version into the appropriate container image tag in the `cryptpad_container_image_tag` variable.

-----------

On a related note, it looks like there's a [2024.12.0](https://github.com/cryptpad/cryptpad/releases/tag/2024.12.0) release now. Have you tried it?

Reply:
Okay, thanx for the reply, it make sense as for the naming.
My apologizies, i didn't catch the git tag, obviously...
I didn't tried to deploy the 2024.12.0 yet, but that's only a matter of time ! 🙂📌

Reply:
You've pushed an update changing `cryptpad_version`, but have not adapted `cryptpad_container_image_tag`, so the role will currently try to pull a missing container image (`docker.io/cryptpad/cryptpad:2024.12.0`) instead of `docker.io/cryptpad/cryptpad:version-2024.12.0`

Reply:
Thank you for pointing it me out,
The docker image isn't available yet with the "YYYY.MM.X" tag.

Well... it seems that i really should not make changes when having a flu...

Reply:
@spantaleev Done !